### DISCUSSION TITLE
Odd discrepancy in XY value between switch and hue lights

### DISCUSSION THREAD

**Discussion by bastiaanterhorst** [basnijholt/adaptive-lighting#354] [Q&A]
Hey there, first of all thanks for this awesome component!

I've added a script in my setup that periodically (every 10 minutes) updates a hue scene with the current values from the adaptive lighting switch. When I turn the lights on, I turn them on to this scene. The idea is that this avoids any bright light in the night, since I've noticed that it takes some time (5-10 seconds) after I turn on the lights for Adaptive Lighting to kick in and adjust the light.

I've got my scenes updating, and when I fetch their values via the Hue API, the ones I get back are the same as what I see in the switch. They all seem to revert to updating xy colours over color temp, but that might be because I have a few lights that don't take ct.

The problem is that when I apply the scene it is consistently a bit warmer than it should be. Meaning after a few seconds there is still quite a substantial adjustment when Adaptive Lighting adjusts the lights.

I've looked where this difference comes from, and it's something weird: both the switch and the scene report the exact same xy color. But when I (using the hue api) fetch the group after the adjustments are done, the xy color is consistently different from what I see reported in the switch!

For example right now, in the switch I see `xy_color: 0.573, 0.389`. And when I load the group, the reported xy color for the lights (controlled by Adaptive Lighting) is: `"xy": [0.4994,0.4153]`.

I really don't get this -- does anyone have any idea where this difference might come from?

**Comment by th3w1zard1** [comment]
Can you set adaptive-lighting to debug mode (see the readme for instructions) and post your config?
